samples/core/guide/autograph.ipynb

```diff
@@ -620,9 +868,7 @@
         "\n",
         "Since writing control flow in AutoGraph is easy, running a training loop in a TensorFlow graph should also be easy.  \n",
         "\n",
-        "<!--TODO(markdaoust) link to examples showing autograph **in** keras models when ready-->\n",
-        "\n",
-        "Important: While this example wraps a `tf.keras.Model` using AutoGraph, `tf.contrib.autograph` is compatible with `tf.keras` and can be used in [Keras custom layers and models](https://tensorflow.org/guide/keras#build_advanced_models). The easiest way is to `@autograph.convert()` the `call` method.\n",
+        "Important: While this example wraps a `tf.keras.Model` using AutoGraph, `tf.contrib.autograph` is compatible with `tf.keras` and can be used in [Keras custom layers and models](https://tensorflow.org/guide/keras#build_advanced_models). \n",
```

Comment:
Another way this could be framed is that AutoGraph is compatible with Keras: AutoGraph code can be used inside Keras models and layers as well; Keras models can also be used in AutoGrahp code, as this example shows.

Reply:
reworded, wdyt?